svelte.config.js

```diff
@@ -1,4 +1,4 @@
-import adapter from '@sveltejs/adapter-vercel'
```

Comment:
Hello @alvinosh 
PR looks nice.
May I know is that package `@sveltejs/adapter-vercel` made any trouble for tauri integration? 
That package used to deploy the app to web. 
https://ark-gallery.vercel.app/
It's not any important thing for this repo, just asking.

Reply:
For tauri to build the app it must be a static application, meaning the files should be generated ahead of time, thats why we cannot use the vercel adapter. If we need to deploy online we can deploy it like a regular static website